Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -34,36 +38,54 @@ namespace O3DE::ProjectManager
         explicit CreateGem(QWidget* parent = nullptr);
         ~CreateGem() = default;
 
+        void ClearFields();
+
+        ProjectManagerScreen GetScreenEnum() override
+        {
+            return ProjectManagerScreen::CreateGem;
+        }
+
     signals:
         void GemCreated(const GemInfo& gemInfo);
 
-    private slots:
+    protected slots:
         void HandleBackButton();
         void HandleNextButton();
+        
+
+    private slots:
         void HandleGemTemplateSelectionTab();
         void HandleGemDetailsTab();
         void HandleGemCreatorDetailsTab();
 
-    private:
-        void LoadButtonsFromGemTemplatePaths(QVBoxLayout* gemSetupLayout);
-        QScrollArea* CreateGemSetupScrollArea();
-        QScrollArea* CreateGemDetailsScrollArea();
-        QScrollArea* CreateGemCreatorScrollArea();
-        QFrame* CreateTabButtonsFrame();
-        QFrame* CreateTabPaneFrame();
+    protected:
         bool ValidateGemTemplateLocation();
         bool ValidateGemDisplayName();
         bool ValidateGemName();
         bool ValidateGemPath();
         bool ValidateFormNotEmpty(FormLineEditWidget* form);
         bool ValidateRepositoryURL();
+        
+        void ProceedToGemDetailsPage();
+        void ProceedToGemCreatorDetailsPage();
+        void ProceedToGemAction();
```

Comment:
These look like there are only used by `CreateGem` - in which case they can be moved to the `private:` section